kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -0,0 +1,11 @@
+{% extends "kolibri/base.html" %}
+{% load learn_tags kolibri_tags %}
+{% block frontend_assets %}
+{{ block.super }}
+{% frontend_assets 'kolibri.plugins.learn.KolibriLearnFrontEnd' %}
+{% content_renderer_frontend_async %}
```

Comment:
Do we expect that the most common scenario is that plugins should be loaded asynchronously?